vortex-serde/src/layouts/read/cache.rs

```diff
@@ -135,15 +140,15 @@ impl LazyDeserializedDType {
 #[derive(Debug)]
 pub struct RelativeLayoutCache {
```

Comment:
This seems like a weird thing to exist?

Reply:
Thsi is the message cache + relative path the dtype thing is weird